posthog/hogql/database/schema/persons.py

```diff
@@ -162,11 +177,56 @@ def to_printed_hogql(self):
         return "raw_persons"
 
 
+# Persons is a lazy table that allows you to insert a where statement inside of the person subselect
+# It pulls any "persons.id in ()" statement inside of the argmax subselect
+# This is useful when executing a query for a large team.
+# It could be useful to set "use_query_cache" if the query is repeated. See `actors_query_runner.py` for an example
```

Comment:
I'm not sure it's clear why you'd want `use_query_cache` if the query is repeated, rather than tweaking the query-runner's `_is_stale()` and using PostHog's caching